changelog/unreleased/issue-3490

```diff
@@ -0,0 +1,10 @@
+Enhancement: Support for specifying file size in read-data-subset

```

Comment:
I'd change this to
```
Support for specifying file size in `check --read-data-subset`
```
That wait it is much more obvious which command is affected by this change.